website/docs/r/cloud_provider_access.markdown

```diff
@@ -6,9 +6,18 @@ description: |-
     Provides a Cloud Provider Access settings resource for registration, authorization, and deauthorization
 ---
 
-# mongodbatlas_cloud_provider_access
+# Cloud Provider Access Configuration Paths
 
-`mongodbatlas_cloud_provider_access` Allows you to register and authorize AWS IAM roles in Atlas.
+The MongoDB Atlas provider offers two paths to perform an authorization for a cloud provider role.
+
+* A single resource path using the `mongodbatlas_cloud_provider_access` that at provision time sets up all the required configuration for a given provider, then with a subsequent update it can perform the authorize of the role.
+
+* A two resource path, consisting of `mongodbatlas_cloud_provider_access_setup` and `mongodbatlas_cloud_provider_access_authorization`. The first resource, mongodbatlas_cloud_provider_access_setup, only generates
```

Comment:
tiny nit of lack of tics for mongodbatlas_cloud_provider_access_setup

Reply:
Hello :) fixed in the last commit